scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -238,8 +238,8 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
 {% endmacro %}
 
 {% macro group_panel() %}
-  <div class="card panel-default" id="case_groups" data-toggle='tooltip' title="Group cases to share user variant assessments like comments, classification and dismissal. A case can be in multiple groups."">
-    <div class="panel-heading">
+  <div class="card panel-default" id="case_groups">
+    <div class="panel-heading" data-toggle='tooltip' title="Group cases to share user variant assessments like comments, classification and dismissal. A case can be in multiple groups.">
```

Comment:
:+1:

Reply:
I'll close this and open a new one, I started the branch from the new release one instead of from master